File: Frontend/myparkingappAdmin/lib/data/dto/request/entry_request.dart
class EntryRequest {
  String objectEncrypt;
  EntryRequest(this.objectEncrypt);
  Map<String, dynamic> toJson() {
    return {
      "objectEncrypt":objectEncrypt
    };
  }

  @override
  String toString() {
    return "objectEncrypt : $objectEncrypt";
  }
}

File: Backend/smartbrain/src/main/java/com/smartparking/smartbrain/dto/request/Entry/EntryRequest.java
package com.smartparking.smartbrain.dto.request.Entry;

import jakarta.validation.constraints.NotEmpty;
import lombok.AccessLevel;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.experimental.FieldDefaults;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
@FieldDefaults(level = AccessLevel.PRIVATE)
public class EntryRequest {
    @NotEmpty(message = "QR code was not be null")
    String objectEncrypt;
}
